tests/unit/EnvTest.php

```diff
@@ -225,14 +268,224 @@ public function testGet_trueTitlecase()
+        
+        // Actual
+        $actual = Env::getArray($varname);
+
+        // Assert
+        $this->assertSame($expected, $actual);
+    }
+    
+    public function testGetArray_0exists()
+    {
+        // Arrange
+        $varname = 'TESTGETARRAY_0EXISTS';
+        $this->putEnv($varname . '=');
+        $expected = [''];
+        
+        // Actual
+        $actual = Env::getArray($varname);
 
         // Assert
         $this->assertSame($expected, $actual);
     }
+    
+    public function testGetArray_1exists()
+    {
+        // Arrange
+        $varname = 'TESTGETARRAY_1EXISTS';
+        $this->putEnv($varname . '=a');
+        $expected = ['a'];
+        
+        // Actual
+        $actual = Env::getArray($varname);
+
+        // Assert
+        $this->assertSame($expected, $actual);
+    }
+    
+    public function testGetArray_multiExists()
+    {
+        // Arrange
+        $varname = 'TESTGETARRAY_MULTIEXISTS';
+        $this->putEnv($varname . '=a,b,c');
+        $expected = ['a', 'b', 'c'];
+        
+        // Actual
+        $actual = Env::getArray($varname);
+
+        // Assert
+        $this->assertSame($expected, $actual);
+    }
+    
+    public function testGetArray_multiEmptyExists()
+    {
+        // Arrange
+        $varname = 'TESTGETARRAY_MULTIEMPTYEXISTS';
+        $this->putEnv($varname . '=a,b,,,c');
+        $expected = ['a', 'b', '', '', 'c'];
+        
+        // Actual
+        $actual = Env::getArray($varname);
+
+        // Assert
+        $this->assertSame($expected, $actual);
+    }
+    
+    public function testGetArray_existsWithDefault()
+    {
+        // Arrange
+        $varname = 'TESTGETARRAY_EXISTSWITHDEFAULT';
+        $this->putEnv($varname . '=a,b,c');
+        $default = ['x', 'y', 'z'];
+        $expected = ['a', 'b', 'c'];
+        
+        // Actual
+        $actual = Env::getArray($varname, $default);
+
+        // Assert
+        $this->assertSame($expected, $actual);
+    }
+    
+    public function testGetArray_notFoundWithDefault()
+    {
+        // Arrange
+        $varname = 'TESTGETARRAY_NOTFOUNDWITHDEFAULT';
+        $default = ['x', 'y', 'z'];
+        $expected = ['x', 'y', 'z'];
+        
+        // Actual
+        $actual = Env::getArray($varname, $default);
+
+        // Assert
+        $this->assertSame($expected, $actual);
+    }
+    
+    public function testGetArray_existsWithInvalidDefaultType()
+    {
+        // Arrange
+        $varname = 'TESTGETARRAY_EXISTSWITHINVALIDDEFAULTTYPE';
+        $this->putEnv($varname . '=a,b,c');
+        $default = 'x,y,z';
+        
+        // Actual
+        $this->expectException(\TypeError::class);
+        Env::getArray($varname, $default);
+
+        // Assert
+        $this->fail("Should have thrown TypeError on default array.");
+    }
+    
+    public function testGetArray_notFoundWithInvalidDefaultType()
+    {
+        // Arrange
+        $varname = 'TESTGETARRAY_EXISTSWITHINVALIDDEFAULTTYPE';
```

Comment:
Looks like this was intended to be `TESTGETARRAY_NOTFOUNDWITHINVALIDDEFAULTTYPE`.